modular_skyrat/modules/morenarcotics/chemistry/reagents/pcp.dm

```diff
@@ -0,0 +1,80 @@
+/atom/movable/screen/fullscreen/color_vision/rage_color
+	color = "#550000"
+
+/datum/reagent/drug/pcp //to an extent this is pretty much just super bath salts
+	name = "PCP"
+	description = "Pure rage put into chemical form."
+	reagent_state = LIQUID
+	color = "#ffea2e"
+	overdose_threshold = 10 //really low overdose to keep people from abusing it too much
+	ph = 8
+	taste_description = "rage"
+	var/datum/brain_trauma/special/psychotic_brawling/bath_salts/pcp_rage
+	var/datum/brain_trauma/special/tenacity/pcp_tenacity
+
+/datum/reagent/drug/pcp/on_mob_metabolize(mob/living/L)
```

Comment:
Use different variable names; its important to use clear and concise naming.

I would recommend you rename your variables to 'mob_living' and 'mob_carbon' for all of these proc.